### PR TITLE
Translate using-addons

### PR DESCRIPTION
**Pull request by MiryangJung**
[Addons](https://storybook.js.org/tutorials/intro-to-storybook/react/en/using-addons/)를 번역하였습니다.

### CODE REVIEW

File: content/intro-to-storybook/react/ko/using-addons.md [hunk 1]
```diff
@@ -85,10 +80,8 @@ LongTitle.args = {
   />
 </video>
 
-[시각적 테스팅(visual testing)](/intro-to-storybook/react/ko/test/)인 경우, 말줄임표에 관한 솔루션이 작동을 멈추었는지에 대한 알림 또한 받을 수 있을 것입니다. 모호한 엣지 케이스들은 테스트 커버리지 없이는 잊혀지기가 쉽습니다!
-
+만약 [시각적 테스팅(visual testing)](/intro-to-storybook/react/ko/test/)이면, 말줄임표 솔루션이 작동 중단되는 경우에도 알림이 표시됩니다. 모호한 edge case들은 테스트 커버리지 없이는 잊혀지기가 쉽습니다!
+`
```
Comment (pak-fuse) [inline content/intro-to-storybook/react/ko/using-addons.md:84]: 백틱을 일부러 넣으셨을까요?

Comment (MiryangJung) [reply content/intro-to-storybook/react/ko/using-addons.md:84]: 😭 😭 😭 😭 😭  오타입니다요 ㅎㅎ

**Comment by cottonpup** [comment]
LGTM!

**Review by cottonpup — APPROVED** [review]
LGTM~

**Review by vanillovin — COMMENTED** [review]
LGTM
